### ISSUE TITLE
Opening a PDF

### ISSUE THREAD

**Issue by lod911** [nextcloud/android#13846]
Hi

I get this error allways when I open a pdf.

Restarr phone and freshly open the app anf then the pff also doesnt help.

Is there a fix?

### Cause of error
```java
Exception in thread "main" java.lang.RuntimeException: Unable to resume activity {com.nextcloud.client/com.owncloud.android.ui.activity.FileDisplayActivity}: java.lang.NullPointerException: Attempt to invoke virtual method 'long java.lang.Long.longValue()' on a null object reference
    at android.app.ActivityThread.performResumeActivity(ActivityThread.java:5244)
    at android.app.ActivityThread.handleResumeActivity(ActivityThread.java:5277)
    at android.app.servertransaction.TransactionExecutor.performLifecycleSequence(TransactionExecutor.java:331)
    at android.app.servertransaction.TransactionExecutor.cycleToPath(TransactionExecutor.java:307)
    at android.app.servertransaction.TransactionExecutor.executeNonLifecycleItem(TransactionExecutor.java:240)
    at android.app.servertransaction.TransactionExecutor.executeTransactionItems(TransactionExecutor.java:152)
    at android.app.servertransaction.TransactionExecutor.execute(TransactionExecutor.java:93)
    at android.app.ActivityThread$H.handleMessage(ActivityThread.java:2595)
    at android.os.Handler.dispatchMessage(Handler.java:107)
    at android.os.Looper.loopOnce(Looper.java:232)
    at android.os.Looper.loop(Looper.java:317)
    at android.app.ActivityThread.main(ActivityThread.java:8592)
    at java.lang.reflect.Method.invoke(Native Method)
    at com.android.internal.os.RuntimeInit$MethodAndArgsCaller.run(RuntimeInit.java:580)
    at com.android.internal.os.ZygoteInit.main(ZygoteInit.java:878)
Caused by: Exception in thread "main" java.lang.NullPointerException: Attempt to invoke virtual method 'long java.lang.Long.longValue()' on a null object reference
    at com.owncloud.android.datamodel.OCFile.isInternalFolderSync(OCFile.java:1058)
    at com.owncloud.android.datamodel.FileDataStorageManager.isPartOfInternalTwoWaySync(FileDataStorageManager.java:2504)
    at com.owncloud.android.ui.fragment.FileDetailFragment.updateFileDetails(FileDetailFragment.java:572)
    at com.owncloud.android.ui.fragment.FileDetailFragment.onViewCreated(FileDetailFragment.java:266)
    at androidx.fragment.app.Fragment.performViewCreated(Fragment.java:3152)
    at androidx.fragment.app.FragmentStateManager.createView(FragmentStateManager.java:608)
    at androidx.fragment.app.FragmentStateManager.moveToExpectedState(FragmentStateManager.java:286)
    at androidx.fragment.app.FragmentManager.executeOpsTogether(FragmentManager.java:2164)
    at androidx.fragment.app.FragmentManager.removeRedundantOperationsAndExecute(FragmentManager.java:2059)
    at androidx.fragment.app.FragmentManager.execPendingActions(FragmentManager.java:2002)
    at androidx.fragment.app.FragmentController.execPendingActions(FragmentController.java:496)
    at androidx.fragment.app.FragmentActivity.onResume(FragmentActivity.java:311)
    at com.owncloud.android.ui.activity.BaseActivity.onResume(BaseActivity.java:97)
    at com.owncloud.android.ui.activity.DrawerActivity.onResume(DrawerActivity.java:1085)
    at com.owncloud.android.ui.activity.FileActivity.onResume(FileActivity.java:246)
    at com.owncloud.android.ui.activity.FileDisplayActivity.onResume(FileDisplayActivity.java:1093)
    at android.app.Instrumentation.callActivityOnResume(Instrumentation.java:1722)
    at android.app.Activity.performResume(Activity.java:9118)
    at android.app.ActivityThread.performResumeActivity(ActivityThread.java:5234)
    at android.app.ActivityThread.handleResumeActivity(ActivityThread.java:5277)
    at android.app.servertransaction.TransactionExecutor.performLifecycleSequence(TransactionExecutor.java:331)
    at android.app.servertransaction.TransactionExecutor.cycleToPath(TransactionExecutor.java:307)
    at android.app.servertransaction.TransactionExecutor.executeNonLifecycleItem(TransactionExecutor.java:240)
    at android.app.servertransaction.TransactionExecutor.executeTransactionItems(TransactionExecutor.java:152)
    at android.app.servertransaction.TransactionExecutor.execute(TransactionExecutor.java:93)
    at android.app.ActivityThread$H.handleMessage(ActivityThread.java:2595)
    at android.os.Handler.dispatchMessage(Handler.java:107)
    at android.os.Looper.loopOnce(Looper.java:232)
    at android.os.Looper.loop(Looper.java:317)
    at android.app.ActivityThread.main(ActivityThread.java:8592)
    at java.lang.reflect.Method.invoke(Native Method)
    at com.android.internal.os.RuntimeInit$MethodAndArgsCaller.run(RuntimeInit.java:580)
    at com.android.internal.os.ZygoteInit.main(ZygoteInit.java:878)

```

### App information
* ID: `com.nextcloud.client`
* Version: `30300090`
* Build flavor: `generic`

### Device information
* Brand: `google`
* Device: `panther`
* Model: `Pixel 7`
* Id: `AP2A.240905.003`
* Product: `panther`

### Firmware
* SDK: `34`
* Release: `14`
* Incremental: `12231197`

**Comment by lod911** [comment]
App Re-Install solved the problem. So I close this issue

**Comment by joshtrichards** [comment]
Yeah we fixed this a little while back. The re-install likely picked up the newer app version.

Original Issue #13502
Fix in #13612 >=3.30.1
